blockexplorer/src/store/validatorsSlice/index.ts

```diff
@@ -4,15 +4,22 @@ import { Status } from "types";
 import { NodeValidator } from "types/node-types";
 import { initialValidatorsStateType, ValidatorType } from "types/store";
 import { loadValidators } from "./utils";
+import moment from 'moment';
+import { LocationNode, NodesPerCountry, NodesPerCity } from '../../types/locationNode';
+import sortBy from 'lodash/sortBy';
 
 let initialState: initialValidatorsStateType = {
   percentageOfActiveValidators: 0,
   numberOfActiveValidators: 0,
   numberOfValidators: 0,
   validatorsLoading: Status.IDLE,
   validators: [],
+  locationNodes: [],
+  nodesPerCountry: [],
+  nodesPerCity: []
 };
 
+//Old Mapping
```

Comment:
i think you forgot to delete the old mapToTableData

Reply:
Done this changes